<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Or_fun_form</class>
 <widget class="QWidget" name="Or_fun_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QPushButton" name="DemoveAll">
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Создание формул ∨</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QListWidget" name="ChoosenFormulas">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="CheckToAll1">
     <property name="text">
      <string>Применить ко всем формулам</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QListWidget" name="listWidget_3">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="MoveAll">
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="CheckBinary">
     <property name="text">
      <string>Бинарная</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="DemoveOne">
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QListWidget" name="ExistedFormulas">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <item>
      <property name="text">
       <string>формула 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>формула 2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="MoveOne">
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="CheckToAll2">
     <property name="text">
      <string>Применить ко всем</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QPushButton" name="OkBtn">
     <property name="text">
      <string>Подтвердить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
